pkgs/development/compilers/gcc/10/default.nix

```diff
@@ -67,6 +67,7 @@ let majorVersion = "10";
       ++ optional langAda ../gnat-cflags.patch
       ++ optional langFortran ../gfortran-driving.patch
       ++ optional (targetPlatform.libc == "musl" && targetPlatform.isPower) ../ppc-musl.patch
+      ++ optional (targetPlatform.isMusl         && targetPlatform.isPower64) ./backport-libgcc-fix-PR97653.patch
```

Comment:
Perhaps it would help to instead default the platform to gcc11? (like x86_64-linux and some others)

Reply:
I'll try that as soon as my current rebuild-the-world finishes.

However even in that case could we still keep the line above?  At least that way if somebody needs gcc10 for some reason it will work.

PS, that line is actually from #170400, which this PR builds on top of.  That PR is ready for review and possible merge; this one is still WIP but getting very close.